node/core/approval-voting/src/lib.rs

```diff
@@ -346,12 +348,15 @@ where
 {
 	fn start(self, ctx: Context) -> SpawnedSubsystem {
 		let backend = DbBackend::new(self.db.clone(), self.db_config);
+		let normal = Normal::new(MALICIOUS_MEAN, MALICIOUS_SDEV);
+		let offset: u64 = normal.sample(&mut rand::thread_rng()).round() as u64;
```

Comment:
I'd recommend adding `3 * STDDEV` which means 99.7% are not edited, but you avoid hickups due to huge outliers.